app/models/store.rb

```diff
@@ -36,6 +36,7 @@ class Store < ApplicationRecord
   has_many :status_crowdsources
   has_many :status_store_owners
   has_many :status_generals
+  has_one :status_general
```

Comment:
Each store can only have one `status_general`, right? Can I remove `has_many status_generals` in favor of `has_one status_general`? (I'll replace existing references)

Reply:
yes that's correct. That's the principle!

Reply:
Yes, the older `status_generals` should be copied to `status_general_histories`.